web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -123,6 +131,74 @@ be edited, its `Revision: X` label will be increased by one, and a note will be
 to the `PDEP-N history` section. This will let readers understand that the PDEP has
 changed and avoid confusion.
 
+## <a id="List-of-issues"></a> List of issues that could have been PDEPs for context
+### Clear examples for potential PDEPs:
+
+- Adding a new parameter to many existing methods, or deprecating one in many places. For example:
+  - The `numeric_only` deprecation ([GH-28900][28900]) affected many methods and could have been a PDEP.
+- Adding a new data type has impact on a variety of places that need to handle the data type.
+  Such wide-ranging impact would require a PDEP. For example:
+  - `Categorical` ([GH-7217][7217], [GH-8074][8074]), `StringDtype` ([GH-8640][8640]), `ArrowDtype`
+- A significant (breaking) change in existing behavior. For example:
+  - Copy/view changes ([GH-36195][36195])
+- Support of new Python features with a wide impact on the project. For example:
+  - Supporting typing within pandas vs. creation of `pandas-stubs` ([GH-43197][43197],
+    [GH-45253][45253])
+- New required dependency.
+- Removing module from the project or splitting it off to a separate repository:
+  - Moving rarely used I/O connectors to a separate repository [GH-28409](28409)
+- Significant changes to contributors' processes are not going to have an impact on users, but
+they do benefit from structured discussion among the contributors. For example:
+  - Changing the build system to meson ([GH-49115][49115])
+
+### Borderline examples:
+Small changes to core functionality, such as `DataFrame` and `Series`, should always be
+considered as a PDEP candidate as it will likely have a big impact on users. But the same types
+of changes in other functionalities would not be good PDEP candidates. That said, any discussion,
+no matter how small the change, which becomes controversial is a PDEP candidate. Consider if more
+attention and/or a formal decision-making process would help. Following are some examples we
+hope can help clarify our meaning here:
+
+- API breaking changes, or discussion thereof, could be a PDEP. For example:
+  - `value_counts` result rename ([GH-49497][49497]). The scope does not justify a PDEP at first, but later a
+    discussion about whether it should be executed as a breaking change or with deprecation
+    emerges, which could benefit from the PDEP process.
+- Adding new methods or parameters to an existing method typically will not require a PDEP for
+  non-core features. For example:
+  - Both `dropna(percentage)` ([GH-35299][35299]), and `Timestamp.normalize()` ([GH-8794][8794])
+    would not have required a PDEP.
+  - On the other hand, `DataFrame.assign()` might. While it is a single method without backwards
+    compatibility concerns, it is also a core feature and the discussion should be highly visible.
+- Deprecating or removing a single method would not require a PDEP in most cases. For example:
+  - That said, `DataFrame.append` ([GH-35407][35407]) is an example of deprecations on core
```

Comment:
perhaps also need to remove the `For example:` to make this scan better if not adding a case where a PDEP is not required.